rst/projections.rst

```diff
@@ -74,31 +74,31 @@ distances are the same:
    155.0
    >>> print( (transform(P2)-transform(P1))[0] )
    155.0
-   >>> print( (transform(P1)-transform(P0))[0] )
+   >>> print( (transform(P3)-transform(P2))[0] )
    155.0
 
 If your data has negative values, you have to use a symmetric log scale (symlog_)
 that is a composition of both a linear and a logarithmic scale. More precisely,
 values around 0 use a linear scale and values outside the vicinity of zero uses
 a logarithmic scale. You can of course specify the extent of the linear zone
-when you set the scale. The logit_ scale is used for values in the range ]0,1[
+when you set the scale. The logit_ scale is used for values in the range [0,1]
```

Comment:
Why this change?

Reply:
Sorry for the late response! Maybe I should use 2 commits here. Change the square brackets because they are wrong because we use the left bracket before the range and the right bracket after the range, namely "[0, 1]", not "]0, 1[".

Reply:
OMG, I'm really sorry, I'll update the change today!

Reply:
No problem. I se you made another PR, I'll merge the new one.